frontend_app_kanban/src/component/Kanban.styl

```diff
@@ -0,0 +1,179 @@
+@import '~tracim_frontend_lib/src/css/Variable.styl'
+
+.kanban__contentpage__statewrapper__kanban
+  display flex
+  flex-direction column
+  height 100%
+
+  &.fullscreen
+    position fixed
+    top headerHeight
+    left 0
+    width 100%
+    height calc(100% - 61px)
+    z-index 1
+    background offWhite
+
+  &__wrapper
+    flex 1
+    overflow auto
+
+    .react-kanban-board
+      overflow-y visible !important
+      overflow-x visible
+
+  &__toolbar
+    background #eff0f1
+
+    button
+      font-size small
+      padding 3px 5px
+      border none
+      border-radius 0
+
+    [disabled]
+      display none
+
+    .iconbutton:not(last-child)
+      border-right 1px solid gray
+
+    &__title
+      display inline-block
+      padding-left 1em
```

Comment:
I would prefer that we use always px to be coherent (same for the other non-px values in this file)